api/question/schema.py

```diff
@@ -68,6 +95,23 @@ def mutate(self, info, question_id, **kwargs):
         return UpdateQuestion(question=exact_question)
 
 
+class DeleteQuestion(graphene.Mutation):
+    class Arguments:
+        question_id = graphene.Int(required=True)
+
+    question = graphene.Field(Question)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, question_id, **kwargs):
```

Comment:
Do we really need to pass **kwargs** as a parameter to this function because I don't see where you are using it as arguments within the function itself?

Reply:
I have resolved this issue although I personally did not work on `DeleteQuestion`, the code was there before, all I did was move the code within the `schema.py` file.

Reply:
cool